docs/extensions/creating-extension.rst

```diff
@@ -53,3 +53,16 @@ to ensure your extension's extraordinary quality.
 .. _`Behat`: http://behat.org/en/latest/
 .. _`phpspec`: http://www.phpspec.net/en/stable/
 .. _`PHPUnit`: https://phpunit.de/
+
+5. Naming conventions
+---------------------
+
+Beside the way how you would like to create a plugin, (based on our skeleton or by your own) there are a few naming conventions that should be followed:
```

Comment:
`Besides the way you are creating plugins (based on our skeleton or on your own), ...`